src/main/resources/wa-task-initiation-ia-asylum.dmn

```diff
@@ -3866,6 +3866,65 @@
           <text>"caseProgression"</text>
         </outputEntry>
       </rule>
+      <rule id="DecisionRule_0jvkz7e">
+        <inputEntry id="UnaryTests_1xlvoo7">
+          <text>"asyncStitchingComplete"</text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_0zet58l">
+          <text>"finalBundling"</text>
```

Comment:
We need to post event state, not pre event state. Please check AdvancedFinalBundlingStateHandler